src/robot/conf/languages.py

```diff
@@ -170,3 +170,265 @@ class Fi(Language):
     arguments = 'Argumentit'
     return_ = 'Paluuarvo'
     bdd_prefixes = {}
+
+class CS(Language):
+    setting_headers = {'Nastavení', 'Nastavení', 'Nastavení', 'Nastavení'}
+    variable_headers = {'Proměnná', 'Proměnné', 'Proměnné', 'Proměnné'}
+    test_case_headers = {'Testovací případ', 'Testovací případy', 'Testovací případy', 'Testovací případy'}
+    task_headers = {'Úloha', 'Úlohy', 'Úlohy', 'Úlohy'}
+    keyword_headers = {'Klíčové slovo', 'Klíčová slova', 'Klíčová slova', 'Klíčová slova'}
+    comment_headers = {'Comment', 'Comments', 'Comments', 'Comments'}
+    library = 'Knihovna'
+    resource = 'Zdroj'
+    variables = 'Proměnná'
+    documentation = 'Dokumentace'
+    metadata = 'Metadata'
+    suite_setup = 'Příprava sady'
+    suite_teardown = 'Ukončení sady'
+    test_setup = 'Příprava testu'
+    test_teardown = 'Ukončení testu'
+    test_template = 'Šablona testu'
+    test_timeout = 'Časový limit testu'
+    force_tags = 'Vynucené štítky'
+    default_tags = 'Výchozí štítky'
+    tags = 'Štítky'
+    setup = 'Příprava'
+    teardown = 'Ukončení'
+    template = 'Šablona'
+    timeout = 'Časový limit'
+    arguments = 'Argumenty'
+    return_ = 'Vrací'
+    bdd_prefixes = {'Pokud', 'Když', 'Pak', 'A', 'Ale'}
+
+class DE(Language):
```

Comment:
Should have two empty rows between classes here and elsewhere.

Reply:
now handled by the newer version of crowdin.py